.rubocop_todo.yml

```diff
@@ -12,12 +12,12 @@ Lint/ParenthesesAsGroupedExpression:
     - 'spec/spacex/dragon_capsules_spec.rb'
     - 'spec/spacex/ships_spec.rb'
 
-# Offense count: 30
+# Offense count: 36
```

Comment:
Does this file need to be committed?

Reply:
I believe so in order to get the rubocop check in travis to pass. We might want to double check with @rodolfobandeira though.

Reply:
Yeah, this is usually added automatically after running `bundle exec rake rubocop gen-auto-config`.

It is usually being used to override rules like large testing blocks. So as long it doesn't override something terrible I'm OK with that. 😄